.github/workflows/tests.yaml

```
@@ -74,7 +74,7 @@ jobs:
         if: steps.cache-npm.outputs.cache-hit != 'true'
         run: npm ci
 
-      - name: Run nodejs tests
+      - name: Run unit tests
```

I don't get it fully because you wrote 
> npm run test:node|browesr` - run tests for given environment. Tests are ordered by unit tests first.

So therefore it runs unit tests and then integration tests or doesn't it?

```suggestion
      - name: Run unit and integration tests
```

---

Thanks, good catch! I started to do some tweaks to the GitHub actions, but then changed the approach a bit so it was not needed, but forgot to revert it.